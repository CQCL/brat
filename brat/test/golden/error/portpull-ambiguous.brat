id(Nat) -> Nat
id = x => x

-- This should fail because there are multiple ports called "a1"
id2(Nat, Nat) -> Nat, Nat
id2 = x, y => (id,id)(a1:x, y)
