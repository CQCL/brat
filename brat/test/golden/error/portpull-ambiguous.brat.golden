Error in test/golden/error/portpull-ambiguous.brat on line 6:
id2 = x, y => (id,id)(a1:x, y)
                     ^^^^^^^^^

  Port a1 is ambiguous in (a1 :: Nat), (a1 :: Nat)

